### PR TITLE
chocolate-doom: Split into separate sub-packages

### PR DESCRIPTION
**Pull request by maikwoehl**
**Summary**

Fixes getsolus/packages#78

**Test Plan**

Install packages and look whether they are showing up correctly in the menu.
Setup programs are starting, the game itself does not start because [you need *wad* files](https://www.chocolate-doom.org/wiki/index.php/FAQ#How_do_I_set_up_Chocolate_Doom_on_my_computer.3F) for that. 

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/c/chocolate-doom/pspec_x86_64.xml [hunk 1]
```diff
@@ -51,20 +33,84 @@
             <Path fileType="doc">/usr/share/doc/chocolate-doom/PHILOSOPHY.md</Path>
             <Path fileType="doc">/usr/share/doc/chocolate-doom/README.Music.md</Path>
             <Path fileType="doc">/usr/share/doc/chocolate-doom/README.md</Path>
+            <Path fileType="data">/usr/share/icons/chocolate-doom.png</Path>
+            <Path fileType="data">/usr/share/icons/chocolate-setup.png</Path>
+            <Path fileType="man">/usr/share/man/man5/chocolate-doom.cfg.5</Path>
+            <Path fileType="man">/usr/share/man/man6/chocolate-doom-setup.6</Path>
+            <Path fileType="man">/usr/share/man/man6/chocolate-doom.6</Path>
+            <Path fileType="man">/usr/share/man/man6/chocolate-server.6</Path>
```
Comment (maikwoehl) [inline packages/c/chocolate-doom/pspec_x86_64.xml:41]: This file should be moved to the server package.

File: packages/c/chocolate-doom/pspec_x86_64.xml [hunk 2]
```diff
@@ -3,46 +3,28 @@
...
-            <Path fileType="data">/usr/share/applications/chocolate-heretic.desktop</Path>
-            <Path fileType="data">/usr/share/applications/chocolate-hexen.desktop</Path>
             <Path fileType="data">/usr/share/applications/chocolate-setup.desktop</Path>
-            <Path fileType="data">/usr/share/applications/chocolate-strife.desktop</Path>
             <Path fileType="data">/usr/share/applications/screensavers/chocolate-doom-screensaver.desktop</Path>
             <Path fileType="data">/usr/share/bash-completion/completions/chocolate-doom</Path>
-            <Path fileType="data">/usr/share/bash-completion/completions/chocolate-heretic</Path>
-            <Path fileType="data">/usr/share/bash-completion/completions/chocolate-hexen</Path>
-            <Path fileType="data">/usr/share/bash-completion/completions/chocolate-strife</Path>
             <Path fileType="doc">/usr/share/doc/chocolate-doom/CMDLINE.doom</Path>
```
Comment (maikwoehl) [inline packages/c/chocolate-doom/pspec_x86_64.xml:28]: The docs may be extracted.

**Review by malfisya — COMMENTED** [review]
This PR changes desktop files and produces new subpackages, I don't think this count as NFC. Good work tho :+1:

**Comment by davidjharder** [comment]
Hey Maik, I see you pushed an update. Is this ready in your opinion?

File: packages/c/chocolate-doom/pspec_x86_64.xml (hunk 2 above)
Comment (maikwoehl) [reply packages/c/chocolate-doom/pspec_x86_64.xml:28]: No need to extract, because every packages has it's own docs

**Comment by maikwoehl** [comment]
> Hey Maik, I see you pushed an update. Is this ready in your opinion?

I will just resolve one more thing and then it should be ready to go.

**Comment by davidjharder** [comment]
OK. I'm going to mark this as "draft". Please change back to "ready for review" when you are done.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**